ubi-ruby-fips/Dockerfile

```diff
@@ -29,3 +29,6 @@ ENV PATH="/usr/pgsql-15/bin:/var/lib/ruby/bin:${PATH}"
 
 ## Install Ruby tools
 RUN gem install --no-document bundler:$BUNDLER_VERSION
+
+## Remove private keys included for Perl IO since we don't use them
+RUN rm -rf /usr/share/doc/perl-IO-Socket-SSL/certs/* 
```

Comment:
Can this actually be moved up and included in the `RUN` block that installs the perl modules? That way we can avoid another layer in the Docker build.